Plugins.cpp

```diff
@@ -52,8 +52,18 @@ PluginInfo::PluginInfo() {
 }
 
 Plugins::Plugins(QObject *p) : QObject(p) {
-	qsSystemPlugins = QLatin1String("plugins/");
-	qsUserPlugins = QLatin1String("./");
+	// Current directory plugins
+	qsCurrentPlugins = QDir::currentPath();
+	// System plugins directory
+	if (QDir("C:/Program Files/Mumble/Plugins").exists())
```

Comment:
Use something like:

    QDir qdDefaultSystemPluginDir("C:/Program Files/Mumble/Plugins");
    if (qdDefaultSystemPluginDir.exists()) {
        qsSystemPlugins = qdDefaultSystemPluginDir.absolutePath();
    }

to avoid dupliacting "C:/Program Files..."